ios/Lift/LiveSessionSensorDataGroupController.swift

```diff
@@ -79,11 +96,17 @@ class SensorDataGroupView : UIView {
                     CGContextStrokeRect(ctx, frame)
                 }
                 y += height + padding
+                if i == 2 && lastEnd <= 0.01 {     // !!!!!!!!!
```

Comment:
Ah, was using that for debugging, forgot to remove.
Also found something that should be fixed, please stand by.